src/vs/workbench/common/views.ts

```diff
@@ -555,6 +555,8 @@ export interface ITreeView extends IDisposable {
 
 	title: string;
 
+	root: ITreeItem;
```

Comment:
Hmm this is in the vs folder. Does this change absolutely need to be made?

Reply:
mmm let me double check